js/navbar-animation.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  const $window = $(window);
+  const $navbar = $('.navbar');
+
+  function hideNavbar() {
+    $navbar.removeClass('show');
+    $navbar.addClass('hide');
+  }
+  function showNavbar() {
+    $navbar.removeClass('hide');
+    $navbar.addClass('show');
+  }
+
+  $(document).ready(() => {
```

Comment:
the order of calling function is wrong .  the js  file you suppose to read from top to button and in order as methods are called .